jobsdb/jobsdb.go

```diff
@@ -861,7 +860,6 @@ func (jd *HandleT) workersAndAuxSetup() {
 
 	jd.logger.Infof("Connected to %s DB", jd.tablePrefix)
 
-	jd.statTableCount = stats.DefaultStats.NewStat(fmt.Sprintf("jobsdb.%s_tables_count", jd.tablePrefix), stats.GaugeType)
```

Comment:
Sorry missed it but any reason why we are not using the group by query in the operator?

Reply:
there is no reason, but changing it is a long process :)

Reply:
In the very beginning, table prefix was part of the metric. Alerts were defined on that metric. Later we defined new metrics and used table prefix as tag. But we never removed the old metric because we had to update dashboards and alert definitions. I think dashboards were updated but alert definitions are not. We need a task to do that. Till that time a TODO in the code makes sense.

BTW, should this PR be seen as a breaking major change? or minor change?